File: 3 КУРС/2 семестр/Учебная практика/JavaProects/app/src/main/java/com/example/voiceassistent/Advice.java
package com.example.voiceassistent;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import java.io.Serializable;
import java.util.List;

public class Advice implements Serializable {

    @SerializedName("slip")
    @Expose
    public adv slip;

    public class adv{
        @SerializedName("advice")
        @Expose
        public String advice;

    }
}

File: 3 КУРС/2 семестр/Учебная практика/JavaProects/app/src/main/java/com/example/voiceassistent/Advice.kt
package com.example.voiceassistent

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Advice : Serializable {
    @JvmField
    @SerializedName("slip")
    @Expose
    var slip: adv? = null

    inner class adv {
        @JvmField
        @SerializedName("advice")
        @Expose
        var advice: String? = null
    }
}